ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -57,9 +60,22 @@ extension PaywallProxy: PaywallViewControllerDelegate {
 
     public func paywallViewController(_ controller: PaywallViewController,
                                       didFinishPurchasingWith customerInfo: CustomerInfo) {
+        self.delegate?.paywallViewController?(controller, didFinishPurchasingWith: customerInfo)
         controller.dismiss(animated: true)
```

Comment:
This is done automatically now so let's remove it: https://github.com/RevenueCat/purchases-ios/pull/3517